cmd/juju/user/login_test.go

```diff
@@ -461,6 +466,8 @@ func (s *LoginCommandSuite) TestLoginErrorSpecifyingUsernameWithOIDC(c *gc.C) {
 	c.Check(stderr, gc.Matches, `ERROR cannot specify a username during login to a controller with OIDC, remove the username and try again\n`)
 }
 
+// This test verifies that login to a controller known to the client
```

Comment:
TestLoginWithOIDCWithNoAccount verifies...

Reply:
I was wondering if we needed the go doc format for tests, but I'll just follow it anyway.